validator/src/main/resources/expected-data-template/otelSDKexpectedHTTPTrace.mustache

```diff
@@ -12,29 +10,26 @@
       "status": 200
     }
   },
+  "metadata": {
+    "default": {
+      "otel.resource.telemetry.sdk.name": "opentelemetry"
+    }
+  },
   "subsegments": [
     {
+      "name": "HTTP GET",
```

Comment:
Can we remove this? The name differs between the SDKS for example the segment mapped by JS is the name of the query "aws.amazon.com".

Reply:
The [OpenTelemetry Specifications on HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md) say that HTTP client spans SHOULD follow this conventation:

> Therefore, HTTP client spans SHOULD be using conservative, low cardinality names formed from the available parameters of an HTTP request, such as "HTTP {METHOD_NAME}"

Only if it is a HTTP server span should it look like the route `/outgoing-http-call`

> In case of HTTP servers, these endpoints are often mapped by the server frameworks to more concise HTTP routes, e.g. /api/users/{user_id}, which are recommended as the low cardinality span names.

`aws.amazon.com` doesn't fit into either of those situations, do you think an issue in the JS SDK would be a better solution?